open_humans/views.py

```diff
@@ -234,7 +237,20 @@ def get_recent_blogposts():
 
     @staticmethod
     def get_recent_activity():
-        recent = ActivityFeed.objects.order_by('-timestamp')[0:12]
+        """
+        Lists the 12 most recent actions by users.
+        """
+        # Here we must use raw sql because the ORM is not quite able to do
+        # the complex 'where' we have here, or at least that's the consensus
+        # in #django on freenode
```

Comment:
Django can't handle taking one object query, then looking up two separate fields from it in an unrelated model, jointly (in this case member_id and project_id).